forte/processors/base/data_augment_processor.py

```diff
@@ -83,6 +83,7 @@ def auto_align_annotations(
             instructed by the "other_entry_policy".
 
         """
+        pass
```

Comment:
well I just learn that we don't need this, but it won't hurt as well. Feel free to merge after travis, I already approved